packages/expo-video/ios/NowPlayingManager.swift

```diff
@@ -0,0 +1,199 @@
+// Copyright 2024-present 650 Industries. All rights reserved.
+
+import Foundation
+import MediaPlayer
+
+/**
+ * A class meant to manage the "NowPlaying" widget based on currently playing players. When multiple players
+ * are present the one that has most recently started playing and will be used as the source of information for the widget.
+ * Paused player will be used as a data source for "NowPlaying" only if no other players are currently playing.
+ */
+class NowPlayingManager {
+  static var shared = {
+    NowPlayingManager()
+  }()
```

Comment:
Static properties behave like globals, so they are evaluated lazily (even without using the `lazy` keyword). This should do the same thing 👇 
```suggestion
  static let shared = NowPlayingManager()
```